crates/ibc/src/lib.rs

```diff
@@ -13,7 +13,7 @@
 #![forbid(unsafe_code)]
 // https://github.com/cosmos/ibc-rs/issues/342
 #![allow(clippy::result_large_err)]
-#![doc(html_root_url = "https://docs.rs/ibc/0.28.0")]
+#![doc(html_root_url = "https://docs.rs/ibc/0.29.0")]
```

Comment:
let's remove this (see guideline https://github.com/rust-lang/api-guidelines/issues/229) and update CONTRIBUTING.md as well

Reply:
[f28f09d](https://github.com/cosmos/ibc-rs/pull/461/commits/f28f09d86fe41ad419cb69975d43bd36b38828f7)